sourcemap/mapper.go

```diff
@@ -4,10 +4,14 @@ import (
 	"fmt"
 	"time"
 
+	"strings"
+
 	"github.com/elastic/beats/libbeat/common"
 	"github.com/elastic/beats/libbeat/logp"
 )
 
+const sourcemapContentSnippetSize = 5
```

Comment:
This could have a huge impact on the used storage so I suggest to make this configurable.

Reply:
you think so? this is in errors - not spans... and only for frontend with sources in uploaded sourcemaps
i didn't want to bloat more the configuration space, but i can make it configurable if you prefer

Reply:
Sourcemapping is generally enabled for frontend endpoints, so it will also be enabled for `transactions`, not only for `errors`. 
If we don't add the config option, it will be hard to give advice to customers in case they want to use sourcemapping but reduce storage.